src/SearchParameters/index.js

```diff
@@ -596,14 +612,19 @@ SearchParameters.validate = function(currentState, parameters) {
       'an error, if it is not, you should first clear the tags with clearTags method.');
   }
 
-  if (currentState.numericFilters && params.numericRefinements && !isEmpty(params.numericRefinements)) {
+  if (
+    currentState.numericFilters &&
+    params.numericRefinements &&
+    !objectHasKeys(params.numericRefinements)
```

Comment:
The condition should be reversed.

```suggestion
    objectHasKeys(params.numericRefinements)
```